eva_sub_cli/etc/eva_schema.json

```diff
@@ -158,7 +158,8 @@
               "Genotyping by array",
               "Curation",
               "Genotyping by sequencing",
-              "Target sequencing"
+              "Target sequencing",
+              "transcriptomics"
```

Comment:
It would be nice to make this case-insensitive, either by lowercasing everything or added a regex pattern in the schema. Unless biovalidator has some particular logic to do this already?

Reply:
I looked into that and there is no way to have a case-insensitive enum in JSON-schema. We could move to regex and list out all the option between or `|` but the error reporting would get difficult.
We could also change the value to lower-case when we convert from spreadsheet to JSON. 
All these require a bit of thought. I've open a [new ticket](https://www.ebi.ac.uk/panda/jira/browse/EVA-3509) for it